content/2016/09/30/cssconf.rst

```diff
@@ -0,0 +1,655 @@
+- Change a ``--gutter`` variable
+  at different viewport sizes,
+  instead of re-defining your gutter properties directly
+- Create property shortcuts with pre-filled default "theme" values
+- Create custom long-hands
+  for changing a single aspect of a short-hand property like ``box-shadow``
+
+You can also use custom properties
+to handle autoprefixing,
+or setting multiple properties at once.
+Setting the global value to ``initial``
+ensures that nothing new is applied by default,
+but any new value
+will be applied to all the properties at once:
+
+.. code:: css
+
+  * {
+    --clip-path: initial;
+    -webkit-clip-path: var(--clip-path);
+    clip-path: var(--clip-path);
+  }
+
+Some custom-property gotchas:
+
+- Properties are case-sensative
+- Don't work well inside the ``url()`` function
+- Can't have an empty value ``:;``
+  but they can have a single space value ``: ;``
+- Values are typed token lists,
+  so you can't do things like ``var(--size)em``
+  to add units to a number
+- Adding units is simple using e.g. ``calc(var(--size) * 1em)``,
+  but there is no good way to remove units —
+  so it is often best to store unitless values,
+  and only add the units when they are needed.
+- Variable definitions (``--my-color``) won't animate,
+  but you can animate properties (``background: var(--my-color)``)
+  that call the variable,
+  and achieve the same outcome.
+
+There's so much more!
+I highly recommend watching the video.
+
+.. _Lea Verou: http://twitter.com/leaverou
+
+
+(I'm not sure about the title)
+------------------------------
+
+.. image:: /static/images/blog/cssconf-2016/ss.jpg
+
+`Sara Soueidan`_
+was scheduled to talk about SVG,
+but talked instead about hacks
+that she has learned to appreciate
+while working on the redesign
+of a major site.
+I didn't catch the new title,
+and found it hard to take notes —
+but this talk is well worth the watch.
+So much good material in here!
+
+.. _Sara Soueidan: http://twitter.com/jen4web
+
+
+The Hateful Weight
+------------------
+
+.. image:: /static/images/blog/cssconf-2016/hh.jpg
+
+`Henri Helvetica`_
+talked about optimizing page and image sizes
+for the web.
+Did you know mp4 videos
+have better performance than gif images?
+Sites like Twitter
+convert your animated gif into mp4 format
+for display.
+
+.. _Henri Helvetica: http://twitter.com/HenriHelvetica
+
+
+Sass Map Magic
+--------------
+
+.. image:: /static/images/blog/cssconf-2016/mia.jpg
+
+I showed a wide range of uses
+for the underused Sass "map" data type —
+from simple site theme configurations,
+to data storage,
+and complex functional programming.
+All the
+`slides <https://oddbooksapp.com/book/sass-map-magic>`_
+are online.
+
+.. _Miriam Suzanne: http://twitter.com/mirisuzanne
+
+
+Webpack and CSS
+---------------
+
+.. image:: /static/images/blog/cssconf-2016/zg.jpg
+
+`Zach Green`_
+walked us through his `Webpack`_ setup.
+I missed most of this,
+recovering from my own talk.
```

Comment:
videos (and slides) will be posted soon!

(when they are, I think we should add links)